products/samsungmobile.md

```diff
@@ -721,10 +1171,960 @@ releases:
+    releaseDate: 2016-10-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy On8"
+    support: false
+    eol: 2019-01-01 # https://doc.samsungmobile.com/SM-J710FN/INS/doc.html
+    releaseDate: 2016-10-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy On7 (2016)"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-10-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy J5 Prime"
+    support: false
+    eol: 2020-10-01 # https://doc.samsungmobile.com/SM-G570F/SER/doc.html
+    releaseDate: 2016-10-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy Note 7"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-09-01 # Approximate to the month and year.
+
+-   releaseCycle: "Z2"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-08-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy Note 7 (USA)"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-08-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy Tab J"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-08-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy J Max"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-08-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy On7 Pro"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-07-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy On5 Pro"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-07-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy J2 Pro (2016)"
+    support: false
+    eol: 2018-08-01 # https://doc.samsungmobile.com/SM-J210F/INS/doc.html
+    releaseDate: 2016-07-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy J2 (2016)"
+    support: false
+    eol: 2018-08-01 # https://doc.samsungmobile.com/SM-J210F/INS/doc.html
+    releaseDate: 2016-07-01 # Approximate to the month and year.
+
+-   releaseCycle: "Z3 Corporate"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-06-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy S7 active"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-06-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy J3 Pro"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-06-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy C7"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-06-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy C5"
+    support: false
+    eol: true # TODO
+    releaseDate: 2016-06-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy A9 Pro (2016)"
+    support: false
+    eol: 2019-02-01 # https://doc.samsungmobile.com/SM-A910F/INS/doc.html
+    releaseDate: 2016-05-01 # Approximate to the month and year.
+
+-   releaseCycle: "Galaxy Tab A 10.1 (2016)"
```

Comment:
Marker for my review.

Pfiou, I do not know how you did to add so much model @Evernow ;)